cdc-accelerator/feature/smsTemplates.js

```diff
@@ -0,0 +1,85 @@
+import SmsConfiguration from '../sap-cdc-toolkit/copyConfig/sms/smsConfiguration.js'
+import fs from 'fs'
+import path from 'path'
+import SiteFeature from './siteFeature.js'
+
+export default class SmsTemplates extends SiteFeature {
+    static FOLDER_OTP = 'SmsTemplates/otp'
```

Comment:
The '/' might not work on windows.
Use path.join to cancatenate paths.
Either way this constant value should be only 'otp'